drivers/ec_host_cmd_periph/ec_host_cmd_periph_shi_npcx.c

```diff
@@ -0,0 +1,992 @@
+/*
+ * Copyright (c) 2022 Google LLC
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#define DT_DRV_COMPAT nuvoton_npcx_shi
+
+#include "ec_host_cmd_periph_shi.h"
+
+#include <zephyr/drivers/clock_control.h>
+#include <zephyr/drivers/ec_host_cmd_periph/ec_host_cmd_periph.h>
+#include <zephyr/drivers/pinctrl.h>
+#include <zephyr/logging/log.h>
+#include <zephyr/mgmt/ec_host_cmd.h>
+#include <zephyr/pm/device.h>
+
+#include <soc_miwu.h>
+
+BUILD_ASSERT(DT_NUM_INST_STATUS_OKAY(DT_DRV_COMPAT) == 1, "Invalid number of NPCX SHI peripherals");
+
+LOG_MODULE_REGISTER(host_cmd_shi_npcx, LOG_LEVEL_DBG);
```

Comment:
Think you meant to use `CONFIG_EC_HC_LOG_LEVEL` here

Reply:
Right, forgot about this...